src/App.js

```diff
@@ -1,24 +1,20 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable import/no-cycle */
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Cities from './components/city';
```

Comment:
I noticed you disabled eslint in this file, it is against microverse policy to disable linters for linters help us write clean code. it would be nice if you enabled eslint.